src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/TopicPartitionWriter.java

```diff
@@ -124,7 +123,7 @@ public void handleRollFile(SourceFile fileDescriptor) {
         }
     }
 
-    private boolean hasStreamingSucceeded(IngestionStatus status) throws URISyntaxException, StorageException {

```

Comment:
This was not throwing exceptions at all so I removed them

Reply:
I will confirm this once , IIRC if we say streaming enabled and then disable streaming on a table / complete DB , will be interesting to see if this RuntimeException is thrown. We'd need to confirm that bad case scenario and then remove this. Will try that out possibly sometime and confirm on this change

Reply:
Both exceptions are checked exceptions, the compiler would most likely complain if we were not handling them, that's why I went ahead and did this change. I can definitely revert if you think this is not safe though

Reply:
You're right. Give me a day or so , Will give this a test by disabling streaming (with it enabled) from KQL and see what happens. You can let it be like this for now!

Reply:
always nice to check . I think this is ok .. anyway the errors can only be thrown now in these calls - getIngestionStatusCollection() ingestFromFile()